android/lib/src/main/java/com/ern/api/impl/navigation/ElectrodeCoreActivity.java

```diff
@@ -0,0 +1,164 @@
+package com.ern.api.impl.navigation;
```

Comment:
Not directly related to this PR, just a general question: I've seen widespread use of `com.ern.*` packages, but I don't think [we own `ern.com`](http://whois.domaintools.com/ern.com). How come we are using this namespace?